packages/clarity-js/types/data.d.ts

```diff
@@ -75,7 +75,14 @@ export const enum Dimension {
     SchemaType = 5,
     ProductBrand = 6,
     ProductAvailability = 7,
-    AuthorName = 8
+    AuthorName = 8,
+    Language = 9,
+    ScreenWidth = 10,
```

Comment:
These should be metric; not dimension. Dimension => string & Metric => number.

Reply:
+1

---
In reply to: [579243873](https://github.com/microsoft/clarity/pull/112#discussion_r579243873) [](ancestors = 579243873)